packages/lesswrong/server/repos/UsersRepo.ts

```diff
@@ -306,8 +306,8 @@ export default class UsersRepo extends AbstractRepo<DbUser> {
             v."userId" = $1
             AND u._id != $1
             AND v."votedAt" > NOW() - INTERVAL '1.5 years'
-            AND v."cancelled" IS FALSE
-    
+            AND v."cancelled" IS NOT TRUE
+
```

Comment:
newline!

Reply:
I'm not adding the newline here, just deleted some extraneous spaces. I think the newline is good for reading clarity because of the UNION ALL

Reply:
(sorry, yeah, the newline is not what I was objecting to, didn't see that it was a different whitespace diff until a second later)

Reply:
not blocking, just prefer to not leave unnecessary whitespace diffs